ddtrace/contrib/elasticsearch/patch.py

```diff
@@ -32,7 +32,7 @@ def _patch(elasticsearch):
         return
     setattr(elasticsearch, '_datadog_patch', True)
     _w(elasticsearch.transport, 'Transport.perform_request', _get_perform_request(elasticsearch))
-    Pin(service=metadata.SERVICE).onto(elasticsearch.transport.Transport)
+    Pin(service=metadata.SERVICE, app=metadata.APP).onto(elasticsearch.transport.Transport)
```

Comment:
is this actually needed?

Reply:
This was mistakenly applied in the previous PR since we are not removing `app` but `app_type` (we still set `app` for pins elsewhere). 

https://github.com/DataDog/dd-trace-py/pull/1150/files#diff-34aefdec68bfc55a5b3206c2df7bec22L35